### PR TITLE
Issue 337 pgp message contained inside signed

### PR DESCRIPTION
**Pull request by michael-volynets**
Issue #337 
Time Spent: 2h

Please merge new mock data first and the rerun tests.
Thanks

### CODE REVIEW

File: test/source/tests/tests/decrypt.ts [hunk 1]
```diff
@@ -191,6 +191,13 @@ export const defineDecryptTests = (testVariant: TestVariant, testWithNewBrowser:
       await InboxPageRecipe.checkDecryptMsg(t, browser, { acctEmail, threadId, expectedContent });
     }));
 
+    ava.default('[standalone] - inbox - encrypted message inside signed', testWithSemaphoredBrowser('compatibility', async (t, browser) => {
+      const inboxPage = await browser.newPage(t, 'chrome/settings/inbox/inbox.htm?acctEmail=flowcrypt.compatibility%40gmail.com&threadId=16f0bfce331ca2fd');
+      const pgpBlock = await inboxPage.getFrame(['pgp_block.htm']);
```
Comment (tomholub) [inline test/source/tests/tests/decrypt.ts]: You have to first await the frame above, look at other usages of `getFrame`

Comment (tomholub) [reply test/source/tests/tests/decrypt.ts]: That will make such tests a little lest flaky, I think

File: test/source/tests/tests/decrypt.ts [hunk 2]
```diff
@@ -191,6 +191,13 @@ export const defineDecryptTests = (testVariant: TestVariant, testWithNewBrowser:
...
+      const content = await pgpBlock.read('#pgp_block');
```
Comment (tomholub) [inline test/source/tests/tests/decrypt.ts]: You have to also await test state "ready"

Comment (tomholub) [reply test/source/tests/tests/decrypt.ts]: image

File: test/source/tests/tests/decrypt.ts (hunk 1 above)
Comment (michael-volynets) [reply test/source/tests/tests/decrypt.ts]: Agree, Thanks!

File: test/source/tests/tests/decrypt.ts (hunk 2 above)
Comment (michael-volynets) [reply test/source/tests/tests/decrypt.ts]: I don't know how I couldn't have noticed this. The test passed when I ran it locally.
Thanks.

**Pull request merged**